examples/ga_peoplecount/README.md

```diff
@@ -23,21 +22,29 @@ const sdk = new HuddlySdk(usbApi);
 async function init() {
   await sdk.init();
 
-  sdk.on('ATTACH', (cameraManager) => {
+  sdk.on('ATTACH', async (cameraManager) => {
+    const autozoomCtl = cameraManager.getAutozoomCtl();
```

Comment:
I don't think there is any reason to use C shorthand for the method names ```getAutozoomControl```

Reply:
Yeah, we can do that.